__init__.py

```diff
@@ -197,6 +197,28 @@ def _handle_switch(self, message):
         self.log.debug("Entity: %s" % entity)
         self.log.debug("Action: %s" % action)
 
+        # Handle turn on/off all intent
+        try:
+            if self.voc_match(entity,"all_lights"):
+                domain = "light"
+            elif self.voc_match(entity,"all_switches"):
+                domain = "switch"
+            else:
+                domain = None
```

Comment:
no handling/response currently for this case (silent failure)

Reply:
should we log / say something, then return?

Reply:
There is nothing wrong, You decide if intent is for all lights or switches. If it not match, then intent is not for all lights/switches and You continue with single entity. (Starting on line 220)